scholly/base/src/global/data.js

```diff
@@ -6,13 +6,38 @@
  * @note: the set environment variables are treated as strings, therefore for the data represented as objects
  * (such as the publications), we need to parse them using JSON.parse().
  */
+import {
+  FAKE_PUBLICATIONS,
+  FAKE_TEAM_INFO,
+  FAKE_TEAM_MEMBERS,
+} from './fakeData';
 
+const env = process.env;
+let TEAM_PUBLICATIONS;
+let TEAM_INFO;
+let TEAM_MEMBERS;
 
-/// The list of publications the team has created for rendering in the publications page
-export const TEAM_PUBLICATIONS = JSON.parse(process.env.REACT_APP_TEAM_PUBLICATIONS);
+if (env.REACT_APP_DEPLOYING === 'true') {
```

Comment:
Prefer calling this environment variable `REACT_APP_DEBUG` so that we can default to fake data if we're running in development mode.

Reply:
Also, we should document this somewhere: i.e. that it is possible to run in Debug mode using this env var, which will run the base app using fake data. Do you reckon writing this process up in a gdocs?

Reply:
Also, add a comment/jsdoc somewhere (e.g. above this line or in the file header) noting that we use fake data if this env var is set.

Reply:
Also, prefer not to do a string comparison for the environment variable. What about just checking if `REACT_APP_DEBUG` is set? https://stackoverflow.com/questions/30047205/how-can-i-check-if-an-environment-variable-is-set-in-node-js